features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/thresholding/FlowThresholding.java

```diff
@@ -158,9 +164,19 @@ private void runTimerTask() {
                     final DeferredGenericTypeResource appResource = new DeferredGenericTypeResource(nodeResource,
                                                                                                     RESOURCE_TYPE_NAME,
                                                                                                     String.format("%s:%s",
-                                                                                                                  application.getKey().iface, // TODO cpape: Find interface name
+                                                                                                                  application.getKey().iface,
```

Comment:
This should use the `ifName`, too. Using the name would stabilize the name and not depend on internal IDs.